src/odb/include/odb/dbTypes.h

```diff
@@ -573,6 +573,11 @@ class dbTechLayerDir
   ///
   operator Value() const { return _value; }
 
+  ///
+  /// Sets the layer-direction.
+  ///
+  void setValue(Value value){ _value = value; }
```

Comment:
operator= would make 
  void setDir(dbTechLayerDir dirIn) { dir.setValue(dirIn.getValue()); }
more natural

Reply:
True. I was just mirroring the style I saw elsewhere in the DB.

Reply:
Oh I see the default `operator=` is sufficient.